lib/OrdinaryDiffEqCore/src/integrators/type.jl

```diff
@@ -105,10 +107,13 @@ mutable struct ODEIntegrator{algType <: Union{OrdinaryDiffEqAlgorithm, DAEAlgori
     tdir::tdirType
     eigen_est::eigenType
     EEst::EEstT
+    # TODO vvv remove these
     qold::QT
     q11::QT
     erracc::QT
     dtacc::tType
+    # TODO ^^^ remove these
+    controller_cache::CC
```

Comment:
Looking at some of the algorithms, we even might want to think moving the controller into the algorithm cache together with some helpers to construct standard controllers.